.github/workflows/rust-msrv.yaml

```diff
@@ -0,0 +1,43 @@
+on:
+  push:
+    branches:
+      - main
+      - dev
+  pull_request:
+    branches:
+      - main
+      - dev
+
+name: MSRV 1.75 Check
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        rust:
+          - 1.75.0 # MSRV
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: Swatinem/rust-cache@v1.2.0
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: ${{ matrix.rust }}
+          override: true
+      # IF NEEDED
+      # - name: Fixes for MSRV
+      #   if: matrix.rust == '1.75.0'
+      #   run: |
+          # cargo update
+          # cargo update -p [CRATE_NAME] --precise [VERSION]
```

Comment:
what is the context for this?

Reply:
Sometimes other crate introduce MSRV changes and we will want to pin those crates into the latest version that supports our MSRV

Reply:
here is an example https://github.com/payjoin/rust-payjoin/blob/master/.github/workflows/rust.yml#L26

Reply:
under which circumstances would that be needed?

The codebase already has a lot of littering in terms of comments saying "todo" and things of that nature. So we should aim for a mindset where we are always aiming for clean code.

I think it's good to be strategize about potential solutions that we might apply for future problems, but maybe we should document these kind of things on different places.

Reply:
Yea good point. deleted.